packages/integration/src/transform.ts

```diff
@@ -62,7 +62,7 @@ export const transform = async ({
       configFile: false,
     });
 
-    if (!result || !result.code) {
+    if (!result || result.code === null) {
```

Comment:
Looks like it can be undefined as well. 

```suggestion
    if (!result || result.code == null) {
```